app/gazebo/tug/plugins/tuginterface/interface/include/tugserver.h

```diff
@@ -0,0 +1,99 @@
+// -*- mode:C++; tab-width:4; c-basic-offset:4; indent-tabs-mode:nil -*-
+/*
+ * Copyright (C) 2015 iCub Facility
+ * Authors: Valentina Vasco
+ * CopyPolicy: Released under the terms of the LGPLv2.1 or any later version, see LGPL.TXT or LGPL3.TXT
+ *
+ */
+
+#ifndef GAZEBO_ASSISTIVEREHAB_TUGSERVER
+#define GAZEBO_ASSISTIVEREHAB_TUGSERVER
+
+#include <yarp/sig/Matrix.h>
+
+#include <gazebo/gazebo.hh>
+#include <gazebo/physics/World.hh>
+#include <gazebo/physics/Actor.hh>
+
+#include <include/utils.h>
+#include <../include/TugInterfaceServer.h>
+#include <../include/Pose.h>
+#include <../include/Animation.h>
```

Comment:
Assuming the availability of relative paths is not that good.
Better off arranging for exposing proper paths instead through CMake.

Reply:
I updated the CMakeLists and removed relative paths.